chain-spec-generator/src/relay_chain_specs.rs

```diff
@@ -0,0 +1,476 @@
+			..Default::default()
+		},
+		grandpa: Default::default(),
+		im_online: Default::default(),
+		authority_discovery: kusama_runtime::AuthorityDiscoveryConfig {
+			keys: vec![],
+			..Default::default()
+		},
+		claims: kusama_runtime::ClaimsConfig { claims: vec![], vesting: vec![] },
+		vesting: kusama_runtime::VestingConfig { vesting: vec![] },
+		treasury: Default::default(),
+		hrmp: Default::default(),
+		configuration: kusama_runtime::ConfigurationConfig {
+			config: default_parachains_host_configuration(),
+		},
+		paras: Default::default(),
+		xcm_pallet: Default::default(),
+		nomination_pools: Default::default(),
+		nis_counterpart_balances: Default::default(),
+	}
+}
+
+fn polkadot_development_config_genesis(
+	wasm_binary: &[u8],
+) -> polkadot_runtime::RuntimeGenesisConfig {
+	polkadot_testnet_genesis(
+		wasm_binary,
+		vec![get_authority_keys_from_seed("Alice")],
+		get_account_id_from_seed::<sr25519::Public>("Alice"),
+		None,
+	)
+}
+
+fn kusama_development_config_genesis(wasm_binary: &[u8]) -> kusama_runtime::RuntimeGenesisConfig {
+	kusama_testnet_genesis(
+		wasm_binary,
+		vec![get_authority_keys_from_seed("Alice")],
+		get_account_id_from_seed::<sr25519::Public>("Alice"),
+		None,
+	)
+}
+
+/// Polkadot development config (single validator Alice)
+pub fn polkadot_development_config() -> Result<Box<dyn ChainSpec>, String> {
+	let wasm_binary =
+		polkadot_runtime::WASM_BINARY.ok_or("Polkadot development wasm not available")?;
+
+	Ok(Box::new(PolkadotChainSpec::from_genesis(
+		"Polakdot Development",
+		"polkadot_dev",
+		ChainType::Development,
+		move || polkadot_development_config_genesis(wasm_binary),
+		vec![],
+		None,
+		Some(DEFAULT_PROTOCOL_ID),
+		None,
+		Some(polkadot_chain_spec_properties()),
+		Default::default(),
+	)))
+}
+
+/// Kusama development config (single validator Alice)
+pub fn kusama_development_config() -> Result<Box<dyn ChainSpec>, String> {
+	let wasm_binary = kusama_runtime::WASM_BINARY.ok_or("Kusama development wasm not available")?;
+
+	Ok(Box::new(KusamaChainSpec::from_genesis(
+		"Kusama Development",
+		"kusama_dev",
+		ChainType::Development,
+		move || kusama_development_config_genesis(wasm_binary),
+		vec![],
+		None,
+		Some(DEFAULT_PROTOCOL_ID),
+		None,
+		None,
+		Default::default(),
+	)))
+}
+
+fn polkadot_local_testnet_genesis(wasm_binary: &[u8]) -> polkadot_runtime::RuntimeGenesisConfig {
+	polkadot_testnet_genesis(
+		wasm_binary,
+		vec![get_authority_keys_from_seed("Alice"), get_authority_keys_from_seed("Bob")],
+		get_account_id_from_seed::<sr25519::Public>("Alice"),
+		None,
+	)
+}
+
+/// Polkadot local testnet config (multivalidator Alice + Bob)
+pub fn polkadot_local_testnet_config() -> Result<Box<dyn ChainSpec>, String> {
+	let wasm_binary =
+		polkadot_runtime::WASM_BINARY.ok_or("Polkadot development wasm not available")?;
+
+	Ok(Box::new(PolkadotChainSpec::from_genesis(
+		"Polkadot Local Testnet",
+		"polkadot_testnet",
```

Comment:
```suggestion
		"polkadot-local",
```
I think this should be aligned with this:
https://github.com/polkadot-fellows/runtimes/pull/127/files#diff-2502728cf4efbc62c3f43ecd31a04613861dcb894cfc75f5ce2389f28f6eecefR25-R31